### ISSUE TITLE
Flaky test on Android GoogleAPI Emulator (CRON)

### ISSUE THREAD

**Issue by web-padawan** [vaadin/vaadin-combo-box#791]
### Description

The keyboard navigation test is failing on Android. Should be most likely skipped at all

```
Android GoogleAPI Emulator ✖ test/keyboard.html » keyboard navigation » selecting items » should select the input field text when navigating down
  expected 3 to deeply equal 0
    <unknown> at   setTimeout at keyboard.html:258:0
```

**Comment by web-padawan** [comment]
Seems to be fixed now after version update done in ae80779
